playbooks/portals-docker-command.yml

```diff
@@ -0,0 +1,55 @@
+- name: Run Docker Command on Skynet Webportal
+  hosts: webportals
+  remote_user: "{{ webportal_user }}"
+  gather_facts: False
+
+  # Target 3 servers at a time.
+  serial: 3
```

Comment:
Follow Up: we should refactor `parallel_deploys` to be a common `num_parallel_hosts` for all playbooks to use.